spark-commons/src/test/scala/za/co/absa/spark/commons/errorhandling/implementations/ErrorMessageArrayTest.scala

```diff
@@ -199,4 +194,34 @@ class ErrorMessageArrayTest extends AnyFunSuite with SparkTestBase {
     assert(result == expected)
 
   }
+
+  test("test return errCol type schema") {
```

Comment:
These 2 tests are valid and work, but they don't really test any computation. It might be better to do something like what you did with tests in `ErrorHandlingFilterRowsTest`, so to have actual DataFrame and perform actual computation :)

Reply:
Well, the thing is that the this method returns the hard coded values which are not formed after any computation in `ErrorHandlingFilterRowsWithErrors` class.

Reply:
Use the same logic here as in `ErrorHandlingFilterRowsWithErrorsTest`(_"errorColumnType should return a BooleanType"_).
Add the column to an empty DataFrame, then ask for it and compare to the `errorColumnType`.